admin/app/components/solidus_admin/ui/pagination/component.html.erb

```diff
@@ -0,0 +1,2 @@
+<div>
+</div>
```

Comment:
Do you think we can squash the commit with these changes with the fourth one (the actual implementation for the component)?